include/openmc/geometry_aux.h

```diff
@@ -7,9 +7,15 @@
 #include <cstdint>
 #include <string>
 #include <vector>
+#include <map>
 
 namespace openmc {
 
+namespace model {
+  extern std::map<int32_t, std::map<int32_t, int32_t>> universe_cell_counts;
+  extern std::map<int32_t, int32_t> universe_level_counts;
```

Comment:
Why not unordered maps? Using regular map will result in O(log N) lookup time rather than O(1).

Reply:
Why not indeed! Will definitely make that switch.